src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -214,6 +214,22 @@ class CommandRequestImpl : public CommandImpl,
   mobile_apis::Result::eType GetMobileResultCode(
       const hmi_apis::Common_Result::eType& hmi_code) const;
 
+  /**
+   * @brief Checks result code from Mobile for single RPC
```

Comment:
@mkorniichuk Do we check result code coming from mobile?

Reply:
@AByzhynar fixed in 5db179eb26efd1a99c0178db3afb9d1e077cb383